lidarrmetadata/api.py

```diff
@@ -62,12 +62,13 @@ def default_route():
     Default route with API information
     :return:
     """
+    print(request.url)
```

Comment:
Assuming you have this in here on purpose? All else looks good, just want to make sure

Reply:
Oops meant to take that out. Will fix.